src/packagedcode/pypi.py

```diff
@@ -187,11 +189,25 @@ def parse_dependencies(location, package):
             package.dependencies = dependencies
 
 
+def file_type(file_name):
+    """
+    Return a renamed filename.
```

Comment:
You are not returning a renamed filename, are you?
Rather you are returning a type of dependency file... so please update your doc accordingly.